spyder/plugins/variableexplorer/widgets/collectionsdelegate.py

```diff
@@ -130,7 +130,7 @@ def createEditor(self, parent, option, index, object_explorer=False):
                     message.format(val_type=val_type, module=module))
                 return
             else:
-                if running_in_mac_app() or is_pynsist():
+                if is_conda_based_app():
```

Comment:
This `if` could also be removed, right?

Reply:
I suppose we should review all the messaging in this file, yes?

Reply:
I'll remove the flow controls, and revise the messaging accordingly.